src/components/vm/hostdevs/hostDevCard.jsx

```diff
@@ -231,8 +231,8 @@ export const VmHostDevCard = ({ vm, nodeDevices, config }) => {
                     <DeleteResourceButton objectId={`${id}-hostdev-${hostdevId}`}
                                           actionName={_("Remove")}
                                           showDialog={() => setDeleteDialogProps({
-                                              objectType: cockpit.format(_("$1 host device"), hostdev.type),
-                                              objectName: "",
+                                              title: _("Remove host device"),
```

Comment:
Here I dropped the `hostdev.type`. Is is necessary or any useful? I think we maybe should add `actionDescription` and show what is going to be removed

Reply:
I'm attending exactly this issue right now at https://github.com/cockpit-project/cockpit-machines/pull/652 where I add actionDescription, so no need to be added at this PR